src/ngraph_mark_for_clustering.cc

```diff
@@ -530,6 +531,7 @@ Status MarkForClustering(Graph* graph,
       type_constraint_map["Reshape"]["T"] = NGraphDTypes();
       type_constraint_map["Reshape"]["Tshape"] = NGraphIndexDTypes();
       type_constraint_map["Rsqrt"]["T"] = NGraphDTypes();
+      type_constraint_map["RsqrtGrad"]["T"] = NGraphDTypes();
```

Comment:
Probably should be realtypes. But did not check. Can you please check and confirm